docs/aboutstudio/sampleapps.mdx

```diff
@@ -2,8 +2,13 @@
 title: Sample Apps
 ---
 
+The following are lists of sample apps using Livepeer Studio features and capabilities:
+
 | Sample App                                                                                                | Description | 
 | -------------------------------------------------------------------------------------------------------------- | ---------- | 
-| Mint NFT                                                        |  Learn how to mint a video NFT                   |
-| Create a Video Player                                                                                               |     Learn how to develop a video player for your application    |
----
\ No newline at end of file
+| [Mint NFT](https://github.com/codentell/video-nft-sample)  |  Learn how to mint a video NFT                   |                                                                             |
+| [Create a Video Player](https://github.com/livepeer/livepeer-jwplayer-demo) |     This is a sample app to showcase how to use Livepeer Studio APIs to create a live stream and play with JWPlayer. The app enables the user to create a stream and play it back via JWPlayer.     |                                        |
+| [WebRTMP](https://github.com/livepeer/webrtmp-sdk ) |  JavaScript SDK for streaming media via RTMP from the Web. Originally designed for Livepeer.com, but can be used for any other service by running your own webrtmp-server. | 
```

Comment:
Designed for Livepeer Studio, this sample app includes instructions for using a JavaScript SDK for streaming media via RTMP from the Web, and can also be implemented for any service running a [webrtmp-server](https://github.com/livepeer/webrtmp-server).